contracts/vaults/RibbonThetaVault.sol

```diff
@@ -536,26 +533,57 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
         address newOption = nextOption;
         require(newOption > PLACEHOLDER_ADDR, "!nextOption");
 
+        uint256 pendingAmount = totalPending();
+        uint256 currentSupply = totalSupply();
+        uint256 currentBalance = assetBalance();
+        uint256 roundStartBalance = currentBalance.sub(pendingAmount);
+        uint16 currentRound = round;
+
+        uint256 singleShare = 10**uint256(_decimals);
+
+        uint256 currentPricePerShare =
+            currentSupply > 0
+                ? singleShare.mul(roundStartBalance).div(currentSupply)
+                : singleShare;
+
+        // After closing the short, if the options expire in-the-money
+        // vault pricePerShare would go down because vault's asset balance decreased.
+        // This ensures that the newly-minted shares do not take on the loss.
+        uint256 mintShares =
+            currentSupply > 0
+                ? pendingAmount.mul(currentSupply).div(roundStartBalance)
```

Comment:
can do `uint256 mintShares = pendingAmount.div(currentPricePerShare).mul(singleShare)`

Reply:
so the ternary operation is more gas effective than no ternary operation + potentially pendingAmount.mul(singleShare).div(singleShare)?

Reply:
I overlooked the ternary, I [removed it](https://github.com/ribbon-finance/ribbon-v2/pull/10/commits/61a5f519ffc12b01f897d37fcdfad355b68f8eb3)